processor/error/event_test.go

```diff
@@ -225,21 +225,48 @@ func TestExplicitGroupingKey(t *testing.T) {
 	}
 
 	for idx, e := range []Event{e1, e2, e3, e4, e5} {
-		assert.Equal(t, groupingKey, e.calcGroupingKey(), "grouping_key mismatch", idx)
+		assert.Equal(t, groupingKey, e.calcGroupingKey(nil), "grouping_key mismatch", idx)
 	}
 }
 
+func TestFramesUsableForGroupingKey(t *testing.T) {
+	frontendConfig := pr.Config{Frontend: true}
+	st1 := m.Stacktrace{
+		m.StacktraceFrame{Filename: "/a/b/c", Lineno: 123},
+		m.StacktraceFrame{Filename: "webpack", Lineno: 77},
+		m.StacktraceFrame{Filename: "~/tmp", Lineno: 45},
+		m.StacktraceFrame{Filename: "tmp", Lineno: 89},
+		m.StacktraceFrame{Filename: "", Lineno: 13},
+		m.StacktraceFrame{Filename: "/webpack", Lineno: 7},
```

Comment:
how about adding another element to this path, so that it wouldn't match https://github.com/elastic/apm-server/pull/482/files#diff-a163837d2241a4bad19d7e4baa4c617bR223 in case the first filter was broken